src/libPMacc/include/algorithms/math/floatMath/exp.tpp

```diff
@@ -56,6 +56,17 @@ struct Log<float>
     }
 };
 
+template<>
+struct Log10<float>
+{
+    typedef float result;
+
+    HDINLINE float operator( )(const float& value )
+    {
+        return ::log10f( value );
```

Comment:
`log10f` is [not avail in C++98](http://www.cplusplus.com/reference/cmath/log10/), please use the overloaded function `log10` on the host side.